script/Arcueid/Day 10/10_00_ARC10_6.txt

```diff
@@ -17,7 +17,7 @@ Staring up at the moon,
 [sha:43a85c634e733f5a4518aa38a5e5308c3e57f88e]{
 -- Page 10020, Offset 26079.
 -- 「けんな、チクショウ……！」
-"―――fuck with me, goddamnit!"
+"―――fuck with me, goddammit!"
```

Comment:
i've always hated that dammit is spelled with two "m"s

Reply:
It's a toss-up
They're both well attested and recognized, so it's just a matter on which side of the fence you fall.
We've settled for "goddamnit", so it's not something that particularly needs correction.